x-pack/plugins/security_solution/common/license/policy_config.ts

```diff
@@ -269,9 +283,12 @@ export const unsetPolicyFeaturesAccordingToLicenseLevel = (
   policy: PolicyConfig,
   license: ILicense | null
 ): PolicyConfig => {
-  if (isAtLeast(license, 'platinum')) {
+  if (isAtLeast(license, 'enterprise')) {
     return policyFactoryWithSupportedFeatures(policy);
   }
+  if (isAtLeast(license, 'platinum')) {
+    return { ...policyFactoryWithSupportedFeatures(policy), global_manifest_version: 'latest' };
+  }
```

Comment:
should we use defaults from `policyFactoryWithoutPaidFeatures();` here?

Reply:
The way I see it - if license is `enterprise` we allow all `policyFactoryWithSupportedFeatures`. If its `platinum` we allow all of the above, however, we do not allow `global_manifest_version` to be anything but latest since its `enterprise` only. Please let me know whether it makes sense.